src/quo/components/animated/pressable.cljs

```diff
@@ -111,21 +115,23 @@
        :min-duration-ms         long-press-duration
        :max-dist                22
        :ref                     long-press-ref}
-      [animated/view {:accessible          true
-                      :accessibility-label accessibility-label}
+      [animated/view
+       {:accessible          true
+        :accessibility-label accessibility-label}
        [gesture-handler/tap-gesture-handler
         (merge gesture-handler
                {:shouldCancelWhenOutside true
-                :wait-for                long-press-ref
-                :enabled                 (boolean (and (or on-press on-long-press on-press-start)
-                                                       (not disabled)))})
+                :wait-for long-press-ref
+                :enabled  (boolean (and (or on-press on-long-press on-press-start)
+                                        (not disabled)))})
```

Comment:
it was better aligned before the change

Reply:
Same comment as I added here https://github.com/status-im/status-mobile/pull/14128#discussion_r1001747569

I think it's related to the heuristic to decide whether to right align or not map keys and let bindings. Maybe we should just disable this heuristic and just force all map keys and let bindings to be right aligned and be done with it. There are consequences for this choice too, but it might be better to reduce unnecessary diff changes.

Reply:
Fixed by set the `:max-variance` to 1000